pages/signup/account-created.tsx

```diff
@@ -0,0 +1,65 @@
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { requireAuthentication } from "@lib/auth";
+import { checkPrivileges } from "@lib/privileges";
+
+import React from "react";
+import { useTranslation } from "next-i18next";
+import { StyledLink } from "@components/globals/StyledLink/StyledLink";
+
+export default function AccountCreated() {
+  const { t, i18n } = useTranslation(["signup"]);
+
+  return (
+    <>
```

Comment:
I wasn't sure if there was enough content to justify a markdown page or a normal html/templated page. I went with html/templated pages.

Reply:
Markdown would have been easier - note for future when having this many strings to localize